src/scripts/main.js

```diff
@@ -1,3 +1,39 @@
 'use strict';
 
-// write code here
+const table = document.querySelector('table');
+const tBody = document.querySelector('tbody');
+const tr = tBody.querySelectorAll('tr');
+
+table.addEventListener('click', (e) => {
+  const header = e.target.closest('th');
+
+  if (header) {
+    const trArr = [...tr];
+
+    if (e.target.textContent === 'Name') {
+      trArr.sort((a, b) =>
+        a.children[0].textContent
+          .localeCompare(b.children[0].textContent));
+    } else if (e.target.textContent === 'Position') {
```

Comment:
You can use [switch...case](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/switch) instead of if...else.
It will make your code cleaner and more understandable